### DISCUSSION TITLE
Change installation host name

### DISCUSSION THREAD

**Discussion by GeorgeVasil** [firefly-iii/firefly-iii#6808] [Firefly III] [answered]
Happy new year to all!

Assuming that I have made a self hosted installation with a domain name georgevasil.ltd, what's the way to change it to georgevasil.xyz ??

I have already change it to my hosting panel with no luck. Do I have to edit smth to the firefly installation??? 

Thank you,

**Comment by JC5** [comment, marked as answer]
Firefly III gets the host name from the web server itself. If that is correct, Firefly III follows.
